systemtests/scripts/run_tests.sh

```diff
@@ -6,4 +6,4 @@ if [[ -n "$TESTCASE" ]]; then
     EXTRA_ARGS="-Dtest=$TESTCASE"
 fi
 
-mvn test -P${PROFILE} -Djava.net.preferIPv4Stack=true -Djansi.force=true -Dstyle.color=always ${EXTRA_ARGS}
+cd .. && mvn test -pl systemtests -P${PROFILE} -am -Djava.net.preferIPv4Stack=true -Djansi.force=true -DfailIfNoTests=false -Dstyle.color=always ${EXTRA_ARGS}
```

Comment:
This mutates current dir. The expectation is that one is still in the same dir afterwards, so cd back into system tests